portfolio/src/main/webapp/script.js

```diff
@@ -102,3 +102,18 @@ $(document).ready(() => {
     fadeDiv('.project');
   });
 });
+
+/**
+ * Adds a greeting to the page.
+ */
+function addGreeting() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    // Convert the HTML string into a document object
+	  let parser = new DOMParser();
+	  let greetingObject = parser.parseFromString(greeting, 'text/html');
```

Comment:
nit: fix spacing

Reply:
@bheff-work I'm not sure what spacing I'm changing here? Should I include a space between initializing the `parser` and the `greetingObject`?